### ISSUE TITLE
Lab 04 data

### ISSUE THREAD

**Issue by AmyTrumpower** [Watts-College/cpp-527-fall-2021#31]
@lecy 
Dr Lecy,

Which year are we using for the IRS data? I went to the IRS website from the link provided in the lab and they have multiple files based on year (2014, 2015, 2016, 2017, 2018, 2019, and 2020)

Thank you,
Amy

**Comment by lecy** [comment]
Use the version archived on GitHub for the lab, please: 

```r
URL <- "https://github.com/DS4PS/cpp-527-spr-2020/blob/master/labs/data/IRS-1023-EZ-MISSIONS.rds?raw=true"
dat <- readRDS(gzcon(url( URL )))
head( dat[ c("orgname","codedef01","mission") ] ) %>% knitr::kable()
```

|orgname                          |codedef01                     |mission                                                                                                                                                                                                                                      |
|:--------------------------------|:-----------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
|NIA PERFORMING ARTS              |Arts, Culture, and Humanities |A community based art organization that inspires, nutures,educates and empower artist and community.                                                                                                                                         |
|THE YOUNG ACTORS GUILD INC       |Arts, Culture, and Humanities |We engage and educate children in the various aspect of theatrical productions, through acting, directing, and stage crew. We produce community theater productions for children as well as educational theater camps and workshops.         |
|RUTH STAGE INC                   |Arts, Culture, and Humanities |Theater performances and performing arts                                                                                                                                                                                                     |
|STRIPLIGHT COMMUNITY THEATRE INC |Arts, Culture, and Humanities |                                                                                                                                                                                                                                             |
|NU BLACK ARTS WEST THEATRE       |Arts, Culture, and Humanities |                                                                                                                                                                                                                                             |
|OLIVE BRANCH THEATRICALS INC     |Arts, Culture, and Humanities |To produce high-quality theater productions for our local community, guiding performers and audience members to a greater appreciation of creativity through the theatrical arts - while leading with respect, organization, accountability. |

**Comment by lecy** [comment]
It is an RDS (R dataset) file. To convert to Excel you can try: 

```r
URL <- "https://github.com/DS4PS/cpp-527-spr-2020/blob/master/labs/data/IRS-1023-EZ-MISSIONS.rds?raw=true"
dat <- readRDS(gzcon(url( URL )))
write.csv( dat, "IRS-1023-EZ-MISSIONS.csv", row.names=F )
```

**Comment by kidistbetter105** [comment]
Hi Dr Lacy !
I installed  =  install.packages("quanteda") and it works . 

But to use IRs file I use the (R dataset) file

URL <- "https://github.com/DS4PS/cpp-527-spr-2020/blob/master/labs/data/IRS-1023-EZ-MISSIONS.rds?raw=true"
dat <- readRDS(gzcon(url( URL )))
write.csv( dat, "IRS-1023-EZ-MISSIONS.csv", row.names=F )

but am having this error.

Error in head(dat[c("orgname", "codedef01", "mission")]) %>% knitr::kable() : 
  could not find function "%>%"

?????

**Comment by AmyTrumpower** [comment]
Hi @kidistbetter105 - I get this one a lot and if I load the dplyr library it seems to resolve.  Maybe that will work for you.

**Comment by lecy** [comment]
Correct, the pipe operator is part of the dplyr library. Thanks Amy!

**Comment by kidistbetter105** [comment]
@AmyTrumpower  It Helps :) Thanks Amy !
